packages/dashmate/test/e2e/lib/manageDockerData.js

```diff
@@ -0,0 +1,78 @@
+const { getConfig } = require('./manageConfig');
+const { SERVICES } = require('./constants/services');
+
+/**
+ * Remove docker volumes
+ * @param {string} configName
+ * @param {object} dockerContainer
+ */
+async function removeVolumes(configName, dockerContainer) {
+  const config = await getConfig(configName);
+  const { COMPOSE_PROJECT_NAME: projectName } = config.toEnvs();
+  const projectVolumeNames = await dockerContainer.getVolumeNames(config.toEnvs());
+
+  projectVolumeNames
+    .map((volumeName) => `${projectName}_${volumeName}`)
+    .map(async (volumeName) => await dockerContainer.getVolume(volumeName).remove());
+}
+
+/**
+ * Remove docker containers
+ * @param {string} configName
+ * @param {object} dockerContainer
+ */
+async function removeContainers(configName, dockerContainer) {
+  const commandOptions = ['--services', '--status=running'];
+
+  const config = await getConfig(configName);
+  const getContainers = await dockerContainer.getContainersList(config.toEnvs(), commandOptions, true);
+  // await dockerContainer.stop(config.toEnvs(), getContainers);
+  await dockerContainer.rm(config.toEnvs(), getContainers);
+}
+
+/**
+ * Check if containers running for group of local nodes
+ * @param {boolean} isRunning
+ * @param {object} dockerContainer
+ */
+async function isGroupServicesRunning(isRunning, dockerContainer) {
```

Comment:
Feel like this function is not really necessary, as `isServiceRunning` already supports group config, or possibly local_seed is breaking it?

Reply:
Unfortunately local_seed is a different case. Agree that both isGroupServicesRunning and isTestnetServicesRunning can be refactored